lib/layout/box_layout.dart

```diff
@@ -221,6 +221,7 @@ class BoxLayoutState extends WidgetState<BoxLayout> with TemplatedWidgetState {
     }
 
     Widget rtn = BoxWrapper(
+        ignoresMargin: true,  
```

Comment:
What happened to the margin? You need to wraps the entire thing around a Padding. See my previous comment about looking at how Image was done. Without that margin doesn't work